src/core/content/connector.js

```diff
@@ -661,11 +669,27 @@ function BaseConnector() {
 	 */
 	this.reactorCallback = null;
 
+	/**
+	 * Used to determine whether or not podcasts should be scrobbled in onStateChanged()
+	 * @type {Boolean}
+	 */
+	let shouldScrobblePodcasts = true;
+	Util.getOption(Options.SCROBBLE_PODCASTS).then((sp) => shouldScrobblePodcasts = sp);
+
 	/**
 	 * Function for all the hard work around detecting and updating state.
 	 */
 	this.stateChangedWorker = () => {
-		if (!this.isScrobblingAllowed()) {
+		/**
```

Comment:
I decided to implement this where `isScrobblingAllowed()` is used as that's how the Google Play connector had podcast checking implemented. It could also be checked as part of the play state check, I believe. Not sure what's preferred.

Reply:
I would pass `Connector.isPodcast` value to the core, and check if a track is a podcast in `Controller`, e.g. in a function where connector state is processed.